sdk/loadtestservice/Azure.ResourceManager.LoadTestService/Azure.ResourceManager.LoadTestService.sln

```diff
@@ -0,0 +1,51 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
please run `eng\scripts\Export-API.ps1 loadtestservice` to generate the api file

Reply:
have added the API file